src/storage/txn/commands/resolve_lock.rs

```diff
@@ -31,8 +32,7 @@ command! {
             /// `txn_status`. `"k4"` will not be affected either, because it doesn't have a non-committed
             /// version.
             txn_status: HashMap<TimeStamp, TimeStamp>,
-            scan_key: Option<Key>,
-            key_locks: Vec<(Key, Lock)>,
+            key_locks: VecDeque<(Key, Lock)>,
```

Comment:
Is the order of resolving locks important?

Reply:
I don't think so.